test/functional/JIT_Test/src/jit/test/vich/JNILocalRef.java

```diff
@@ -53,38 +53,38 @@ public JNILocalRef() {
 	@Test(groups = { "level.sanity","component.jit" })
 	public void testJNILocalRef()
 	{
-		Object o1 = new Integer(0);
-		Object o2 = new Integer(0);
-		Object o3 = new Integer(0);
-		Object o4 = new Integer(0);
-		Object o5 = new Integer(0);
-		Object o6 = new Integer(0);
-		Object o7 = new Integer(0);
-		Object o8 = new Integer(0);
-		Object o9 = new Integer(0);
-		Object o10 = new Integer(0);
-		Object o11 = new Integer(0);
-		Object o12 = new Integer(0);
-		Object o13 = new Integer(0);
-		Object o14 = new Integer(0);
-		Object o15 = new Integer(0);
-		Object o16 = new Integer(0);
-		Object o17 = new Integer(0);
-		Object o18 = new Integer(0);
-		Object o19 = new Integer(0);
-		Object o20 = new Integer(0);
-		Object o21 = new Integer(0);
-		Object o22 = new Integer(0);
-		Object o23 = new Integer(0);
-		Object o24 = new Integer(0);
-		Object o25 = new Integer(0);
-		Object o26 = new Integer(0);
-		Object o27 = new Integer(0);
-		Object o28 = new Integer(0);
-		Object o29 = new Integer(0);
-		Object o30 = new Integer(0);
-		Object o31 = new Integer(0);
-		Object o32 = new Integer(0);
+		Object o1 = Integer.valueOf(0);
+		Object o2 = Integer.valueOf(0);
```

Comment:
Any requirement for these to be unique objects?  valueOf will use the Integer cache where `new Integer` wouldn't

Reply:
This probably does change the sense of the test, but this test is pretty useless these days, and continues to pass with these changes.

Reply:
Updated to use unique objects.

Reply:
Changing them to `new Object()` is potentially giving our selves more work in the future considering that Object may become an abstract class. For these sort of things where we need a ref type but dont really care what kind we should just define a static inner class.

Reply:
Updated.